tests/test_xopen.py

```diff
@@ -690,6 +727,8 @@ def test_xopen_read_from_pipe(ext, threads):
 
 @pytest.mark.parametrize("threads", (0, 1))
 def test_xopen_write_to_pipe(threads, ext):
+    if ext == ".lz4" and shutil.which("lz4") is None:
+        pytest.skip("lz4 not installed")
```

Comment:
These clauses will also skip when lz4 is not installed but python-lz4 is. That is not ideal.